WORKSPACE

```diff
@@ -66,6 +66,32 @@ rbe_autoconfig(
     ),
 )
 
+git_repository(
```

Comment:
Dependencies need to go in `bazel/grpc_deps.bzl`, so that whomever imports us can grab the same list of dependencies, just like what you're doing below with the `apple_rules_dependencies()`